src/plugins/workspace/public/plugin.ts

```diff
@@ -33,7 +33,7 @@ export class WorkspacePlugin implements Plugin<{}, {}, {}> {
     }
   }
 
-  private getWorkspaceIdFromURL(basePath?: string): string | null {
+  private getWorkspaceIdFromURL(basePath: string): string | null {
```

Comment:
Thanks for this change~

Nit:
Since `getWorkspaceIdFromURL` is a private function and have only one line to call another function.

What if we get rid of this `getWorkspaceIdFromURL(basePath: string)` function, and directly call `getWorkspaceIdFromUrl(window.location.href, basePath)` instead?

Thanks

Reply:
Sure, will do that.